regression/scripts/solar_network/solar_network.py

```diff
@@ -9,7 +9,8 @@
 

```

Comment:
I added it because I needed to test all of the remaining plot functions (solar plots etc). I did not want to create another regression etc for it so I thought it best to create a mission.

Reply:
Ok, but run it with 4 control points at least instead of 64